packages/@webex/plugin-meetings/test/unit/spec/breakouts/index.ts

```diff
@@ -703,5 +736,15 @@ describe('plugin-meetings', () => {
         );
       });
     });
+
+    describe('#assign', () => {
+      it('assign attendee to a breakout session', async () => {
+        breakouts.assign = sinon.stub().returns(Promise.resolve('ASSIGN_RETURN_VALUE'));
+        const params = {assigned: []};
```

Comment:
there needs to be a tests for the following:
1) emails but no member ids
2) member ids but no emails